scripts/kconfig/kconfig.py

```diff
@@ -97,6 +98,28 @@ def main():
     kconf.write_autoconf(args.autoconf)
 
 
+# Message printed when a promptless symbol is assigned (and doesn't get the
+# assigned value)
+PROMPTLESS_HINT = """
+This symbol has no prompt, meaning direct assignments have no effect on it. It
```

Comment:
We don't use the term "direct assignment" in our documentation on application configuration.

Perhaps we can mention the term "configuration file" as that is mentioned.

So something like 

> This symbol has no prompt, meaning direct assignments by configuration files have no effect.

Reply:
Alternatively we could update the docs ...